src/phpcs/Production/Sniffs/Classes/UseNamespacePathSniff.php

```diff
@@ -14,22 +20,18 @@ class Production_Sniffs_Classes_UseNamespacePathSniff implements PHP_CodeSniffer
      */
     public $supportedTokenizers = ['PHP'];
 
-    /**
-     * @var string[]
-     */
+    /** @var string[] */
     private $usePaths = [];
 
-    /**
-     * @var string[]
-     */
+    /** @var string[] */
     private $wantedAnnotations = [
         '@var',
         '@param',
         '@throws',
         '@return',
     ];
 
-    /** @var PHP_CodeSniffer_File $file */
+    /** @var File */
```

Comment:
For local variables, the annotation always also needs to have the variable name.

Reply:
As far as I know this is only necessary if the line of code contains multiple references to variables. I think this may never be the case for properties of classes, but most certainly in this instance it's not.

Reply:
According to http://docs.phpdoc.org/references/phpdoc/tags/var.html , the name must always be set (except for properties).

Reply:
Thanks for the link, but in this case it is a property.

Reply:
Oh, I missed that. In that case, there indeed should not be the property name.